embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/capture/startup/StartupServiceImpl.kt

```diff
@@ -14,17 +22,25 @@ internal class StartupServiceImpl(
 
     override fun setSdkStartupInfo(startTimeMs: Long, endTimeMs: Long) {
         if (sdkStartupDurationMs == null) {
-            spanService.recordCompletedSpan(
-                name = "sdk-init",
-                startTimeMs = startTimeMs,
-                endTimeMs = endTimeMs
-            )
+            backgroundWorker.submit {
```

Comment:
Why is a backgroundworker required here? My first thought is the span service should take care of any perf penalty (if there is any from making this call)

Reply:
And this should probably live in a different commit/PR because it's not directly related to the POC 😅

Reply:
I would lean towards just recording this on the calling thread as the background worker performs operations such as I/O - I'm wary that could throw off the accuracy of the span. I guess that depends on whether we record any other spans during SDK init. If we do, then we're going to pay that penalty anyway. This isn't blocking feedback so I'll leave it up to you

Reply:
I restored the old behaviour and will improve this in the next release.